tests/integration/ha/test_backups.py

```diff
@@ -310,40 +316,14 @@ async def test_restore_cluster_after_app_destroyed(ops_test: OpsTest) -> None:
         timeout=1400,
         idle_period=IDLE_PERIOD,
     )
-
-    units = await get_application_unit_ids_ips(ops_test, app=app)
-    leader_id = await get_leader_unit_id(ops_test, app)
-
-    action = await run_action(ops_test, leader_id, "restore", params={"backup-id": 1})
-    logger.info(f"restore output: {action}")
-    assert action.status == "completed"
-
-    # index document
-    doc_id = TEST_BACKUP_DOC_ID
-    # check that the doc can be retrieved from any node
-    logger.info("Test backup index: searching")
-    for u_id, u_ip in units.items():
-        docs = await search(
-            ops_test,
-            app,
-            u_ip,
-            TEST_BACKUP_INDEX,
-            query={"query": {"term": {"_id": doc_id}}},
-            preference="_only_local",
-        )
-        # Validate the index and document are present
-        assert len(docs) == 1
-        assert docs[0]["_source"] == default_doc(TEST_BACKUP_INDEX, doc_id)
```

Comment:
Sry I may be getting lost... Where is this index check thing gone and if it's gone completely, why don't we need it anymore?

Reply:
@juditnovak it is not gone, but this check is done in the _backup and _restore tests already.